beacon-chain/rpc/metrics.go

```diff
@@ -0,0 +1,31 @@
+package rpc
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var (
+	httpRequestLatency = promauto.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Name:    "http_request_latency_seconds",
+			Help:    "Latency of HTTP requests in seconds",
+			Buckets: []float64{0.001, 0.01, 0.025, 0.1, 0.25, 1, 2.5, 10},
+		},
+		[]string{"endpoint", "code", "method"},
+	)
+	httpRequestCount = promauto.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "http_request_count",
+			Help: "Number of HTTP requests",
+		},
+		[]string{"endpoint", "code", "method"},
+	)
+	errorCount = promauto.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "http_error_count",
+			Help: "The total number of HTTP errors for requests to beacon node",
+		},
+		[]string{"endpoint", "method", "status"},
```

Comment:
What's the difference between `code` and `status` labels? The two other metrics use `code`.

Reply:
Replaced it with code and also rearranged the order as "endpoint, code, method" same as other metrics.